src/main/java/ch/bfh/bti7081/presenter/SeminarPresenter.java

```diff
@@ -20,15 +22,42 @@
     public SeminarPresenter() {
     }
 
+
     public List<SeminarCategory> getSeminarCategories() {
         return seminarCategoryManager.getSeminarCategories();
     }
 
-    public List<Seminar> getSeminaries() {
+    private List<Seminar> getSeminaries() {
         return seminarManager.getSeminaries();
     }
 
-    public List<Seminar> getFilteredSeminaries(SeminarFilter filter) {
+    private List<Seminar> getFilteredSeminaries(SeminarFilter filter) {
         return seminarManager.getFilteredSeminars(filter);
     }
+    public List<SeminarDTO> getSeminarDtos() throws Exception{
+        return convertModelsToDtos(getSeminaries());
+    }
+    public List<SeminarDTO> getFilteredSeminarDtos(SeminarFilter filter) throws Exception{
+
+        return convertModelsToDtos(getFilteredSeminaries(filter));
+    }
+
+    private List<SeminarDTO> convertModelsToDtos(List<Seminar> seminaries) throws Exception{
+        List<SeminarDTO> seminarDtos = new ArrayList<>();
+        for (Seminar modelObject : seminaries) {
+            SeminarDTO seminarDTO = new SeminarDTO();
+            seminarDTO.setCategory(modelObject.getCategory().getName());
+            seminarDTO.setDate(modelObject.getDate().toLocalDate());
+            seminarDTO.setDescription(modelObject.getDescription());
+            seminarDTO.setHouseNumber(modelObject.getHouseNumber());
+            seminarDTO.setLocation(modelObject.getLocation());
+            seminarDTO.setPlz(modelObject.getPlz().doubleValue());
+            seminarDTO.setStreet(modelObject.getStreet());
+            seminarDTO.setTime(modelObject.getDate().toLocalTime());
+            seminarDTO.setTitle(modelObject.getTitle());
+            seminarDTO.setUrl(modelObject.getUrl());
+            seminarDtos.add(seminarDTO);
```

Comment:
ID fäut?

Reply:
Ist m.E. nicht relevant und existiert im DTO doch gar nicht? Ist das DTO nicht dazu da, dass die Daten in der View vom Backend entkoppelt sind?

Reply:
Ist sehr wohl relevant, weil aufgrund dessen unterschieden wird ob es den User schon gibt oder nicht... Ausser das DTO wird beim Login/Logout/neuen Benutzer erstellen NICHT verwendet

Reply:
Mmh, es geht doch hier um Seminare? (•‿•)

Reply:
Geht definitiv um Seminare ;-). Im seminardto ist keine ID vorhanden und diese wird auch nirgends verwendet.

Reply:
Also werden wir niemals Seminare editieren können?

Reply:
glichi diskussion wie bim user. wemer ds implementiere muesme a bestehende sache sehr viu ändere, i denke dass ds es thema füre nächst sprint isch u de de umsetzt wo de task gfasset het...